### ISSUE TITLE
show a message instead of failing when using the conversion-service api client

### ISSUE THREAD

**Issue by hixi** [geometalab/osmaxx-frontend#389]
For the status and other requests going to the conversion-service, generate a default message when it is not reachable or returns in an erroneous state. After discussion, we agreed that this should be done on the `osmaxx.excerptexport.services.conversion_api_client.ConversionApiClient` or even further up (caller side), out of the following reasons:
- simpler to create defaults in python than json objects, whereof we often only need one entry
- easier to define a timeout for unresponsiveness in specific cases
- visibility of exception handling is better
- no guessing for what kind of case it covers

**Comment by hixi** [comment]
@das-g isn't this issue obsolete now?

**Comment by das-g** [comment]
> isn't this issue obsolete now?

I don't think so. If anything, we would get more<sup>*</sup> failures now, as we raise (or let raise by not excepting) more errors in the client than we used to. This will still lead to 5xx, but not redacting the original errors will allow better diagnostics than with the previous state.

Though, this issue is somewhat broad and open-ended. So we might just close it and open new ones when errors occur that we think should be handled more gracefully.

<sup>*</sup>That is, in more cases. Dunno about the frequency, as we make more calls than we used to in some areas and less than we used to in others.
